contrib/kind.sh

```diff
@@ -171,7 +171,7 @@ OVN_IMAGE=${OVN_IMAGE:-local}
 # NOTE: Upstream KIND IPv6 masks are different (currently rejected by ParseClusterSubnetEntries()):
 #  Upstream - NET_CIDR_IPV6=fd00:10:244::/64 SVC_CIDR_IPV6=fd00:10:96::/112
 NET_CIDR_IPV4=${NET_CIDR_IPV4:-10.244.0.0/16}
-SVC_CIDR_IPV4=${SVC_CIDR_IPV4:-10.96.0.0/12}
+SVC_CIDR_IPV4=${SVC_CIDR_IPV4:-10.96.0.0/16}
 NET_CIDR_IPV6=${NET_CIDR_IPV6:-fd00:10:244::/48}
 SVC_CIDR_IPV6=${SVC_CIDR_IPV6:-fd00:10:96::/64}
```

Comment:
want to fix this one too while you're here? (Not sure how this is even working... something in kind must rewrite it?)

Reply:
I was going to do it but the comment scared me
```
(currently rejected by ParseClusterSubnetEntries())
```
what does it mean?

Reply:
duh, kind is not using those paremeters.
Those are only used for ovn

Reply:
done